roles/boot_disk/tasks/nokia.yml

```diff
@@ -0,0 +1,84 @@
+---
+- name: UnMount Live ISO, Boot into HDD (Nokia servers)
+  block:
+    - name: Nokia Power ON
+      community.general.redfish_command:
+        category: Systems
+        command: PowerOn
+        baseuri: "{{ hostvars[item]['bmc_address'] }}"
+        username: "{{ hostvars[item]['bmc_user'] }}"
+        password: "{{ hostvars[item]['bmc_password'] }}"
+
+    - name: Eject ISO
+      block:
+      - name: Check if there is a ISO mounted on the Nokia
+        uri:
+          url: "https://{{ hostvars[item]['bmc_address'] }}/redfish/v1/Managers/1/VirtualMedia/CD1"
+          user: "{{ hostvars[item]['bmc_user'] }}"
+          password: "{{ hostvars[item]['bmc_password'] }}"
+          method: GET
+          validate_certs: no
+          force_basic_auth: yes
+          return_content: yes
+        register: cd1_contents
+
+      - name: Unmount Nokia ISO (VirtualMedia)
+        uri:
+          url: "https://{{ hostvars[item]['bmc_address'] }}/redfish/v1/Managers/1/VirtualMedia/CD1/Actions/VirtualMedia.EjectMedia"
+          user: "{{ hostvars[item]['bmc_user'] }}"
+          password: "{{ hostvars[item]['bmc_password'] }}"
+          method: POST
+          headers:
+            content-type: application/json
+            Accept: application/json
+          body: {}
+          body_format: json
+          validate_certs: no
+          force_basic_auth: yes
+          return_content: yes
+        when: cd1_contents.json.Inserted | bool == True
+      rescue:
+        - name: Unmount Nokia ISO (ISO Config)
+          uri:
+            url: "https://{{ hostvars[item]['bmc_address'] }}/redfish/v1/Managers/1/VM1/CfgCD/Actions/IsoConfig.UnMount"
```

Comment:
This doesn't match the fall back URL in `boot_iso`, I suggest we drop the fall back as you said it isn't used.